### ISSUE TITLE
Can't create custom filament since updating to version Bambu_Studio_win-v01.10.01.50 

### ISSUE THREAD

**Issue by Ohman-BigO** [bambulab/BambuStudio#5358]
### Bambu Studio Version

01.10.01.50 

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11

### Additional system information

CPU:13th Gen Intel Core i9-13900H 16GB

### Printer

Bambu Labs P1S

### How to reproduce

Select filament to use - Custom Filament - Create new 
No printers are shown and I can't select a Filament Preset

### Actual results

Error that no printer was selected, no filament is created

![Screenshot 2024-11-17 114508](https://github.com/user-attachments/assets/5a733720-fb09-4c37-b4b9-a9915cd137e8)

### Expected results

A custom filament should be added for use

### Project file & Debug log uploads

No Project file, because it doesn't matter what I try to print, I can't create a filament.

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by JohnSmithCYM** [comment]
![20241122093157_rec_](https://github.com/user-attachments/assets/e8b44522-7843-4030-9f79-a4a48d62618e)
The gif shows the operation that I created a filament like your information with 1.10.1.50. It seems works well, so is there anything I missed?

**Comment by ttt888zzz** [comment]
HI [Ohman-BigO](https://github.com/Ohman-BigO)：Please have a try following the steps.![20241122101035_rec_](https://github.com/user-attachments/assets/0f99999a-9b24-4334-b3a1-6fb01075b820)

**Comment by Ohman-BigO** [comment]
OMG I found the problem! Because I have no PLA-CF filaments created, it
doesn't have anything to show in the list. When I select PLA it finds some
already created and makes menu entries. In my opinion, this empty menu is
confusing and should at least show Bambu filaments. A menu that doesn't
open and doesn't bring an error appears broken.

Thanks for the help, close the ticket please.

Kind regards
Steve

**Comment by Ohman-BigO** [comment]
Hi John, thanks for your answer, at least it showed me that the problem is
local.

I tried it again and still had an empty menu with PLA-CF selected. I chose
PLA and it showed me the other PLA filaments I had created, then it was
clear.

A blank menu is a bad thing, I would suggest that if no filament in this
class has been created, that it goes back to the standard filaments and
takes one from there.

Again, thank you for your assistance.

Kind regards
Steve
